src/api/tokenList/Faucet_info.md

```diff
@@ -1,51 +1,61 @@
 # Ether, Tether, TrueUSD, USD Coin, Paxos, Gemini, Dai
 
 ## Wrapped Ether
-* [https://weth.io](https://weth.io)
-* Wrapper of Ether to make it ERC-20 compliant
-* **Rinkeby faucet**: [https://faucet.rinkeby.io/](https://faucet.rinkeby.io)
 
-## Tether:
-* [https://tether.to](https://tether.to)
-* Fiat enabled collateralized stable coin, that is backed by the most popular fiat currency, USD (US Dollar) in a 1:1 ratio
-* **Rinkeby faucet**: contract created with Team's shared account 0xf85D1a0E1b71e72013Db51139f285C6d5356B712.
-Use it to mint new tokens
+- [https://weth.io](https://weth.io)
+- Wrapper of Ether to make it ERC-20 compliant
+- **Rinkeby faucet**: [https://faucet.rinkeby.io/](https://faucet.rinkeby.io)
```

Comment:
You don't need `[url](url)` when it's the same url. https://faucet.rinkeby.io/ would have been enough

Just an fyi, no need to change

Reply:
Nice to know!

Reply:
True, although not all MD implementation support that. I think Github does, so I agree it should be enough in this case.

Reply:
I only `fixed` it because VScode MD linter complained about it.